bucket/goodbyedpi.json

```diff
@@ -0,0 +1,29 @@
+{
+    "version": "0.2.1",
+    "description": "Bypass Deep Packet Inspection systems found in many Internet Service Providers which block access to certain websites.",
+    "homepage": "https://github.com/ValdikSS/GoodbyeDPI",
+    "license": "Apache-2.0",
+    "url": "https://github.com/ValdikSS/GoodbyeDPI/releases/download/0.2.1/goodbyedpi-0.2.1.zip",
+    "hash": "acb0568cfc184cdd52c235c9ccc51a38fd78eb57564dcf7dce5188b9ad93f391",
+    "extract_dir": "goodbyedpi-0.2.1",
+    "bin": "goodbyedpi.exe",
+    "architecture": {
+        "64bit": {
+            "pre_install": [
+                "Move-Item \"$dir\\x86_64\\*\" \"$dir\\\"",
+                "Remove-Item \"$dir\\x86\", \"$dir\\x86_64\" -Force -Recurse"
+            ]
+        },
+        "32bit": {
+            "pre_install": [
+                "Move-Item \"$dir\\x86\\*\" \"$dir\\\"",
+                "Remove-Item \"$dir\\x86\", \"$dir\\x86_64\" -Force -Recurse"
```

Comment:
It's not necessary, but since this line is common, should we move it outside architecture block (post_install)?

Reply:
I don't think we should do this. It reduces `pre_install` by a little bit, but introduces `post_install`. This actually increases the complexity IMO.

Reply:
Okay, sounds good